data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -38,6 +38,20 @@ OPENSHIFT_HYPERKUBE_IMAGE=$(podman run --quiet --rm ${release} image hyperkube)
 
 CLUSTER_BOOTSTRAP_IMAGE=$(podman run --quiet --rm ${release} image cluster-bootstrap)
 
+# Verify at this point we have no images from docker.io or gcr.io
+repos=$(mktemp --suffix='bootkube')
+podman images --sort repository --format '{{"{{"}}.Repository {{"}}"}}' | sort -u > ${repos}
+if grep -E '(docker\.io|gcr\.io)/' ${repos}; then
+  echo 'Disallowed registries found!' 1>&2
+  exit 1
+fi
+
+# Now, as early as possible we replace the pause image and restart crio to use it, to ensure
+# that we're using the pause image from our payload just like the primary cluster.
+# Nothing should have created a pod yet.
+sed -i -e 's,pause_image *=.*,pause_image = "'${MACHINE_CONFIG_INFRA_IMAGE}'",' /etc/crio/crio.conf
```

Comment:
Can you include in the commit message, an example cri-o conf that you are writing the sed against.

Reply:
I have a local change to the comments that mentions that this is based against the MCO's [crio.yaml](https://github.com/openshift/machine-config-operator/blob/master/templates/master/01-master-container-runtime/_base/files/crio.yaml)  (and indeed I had forgotten to also change `pause_command`, done now locally)